Modules/_cursesmodule.c

```diff
@@ -4558,10 +4558,6 @@ PyInit__curses(void)
     PyObject *m, *d, *v, *c_api_object;
     static void *PyCurses_API[PyCurses_API_pointers];
 
-    /* Initialize object type */
-    if (PyType_Ready(&PyCursesWindow_Type) < 0)
-        return NULL;
-
```

Comment:
It looks wrong to me to expose PyCursesWindow_Type through PyCurses_API before it is made explicitly ready. Maybe just keep the explicitly PyType_Ready() call.

Reply:
Okay, I 've reverted the change of _cursesmodules.

Reply:
I still consider that it's good to use PyModule_AddType(), since PyModule_AddObject() is an ugly API and it's even misused in this file. I just asked to keep the PyType_Ready() call.

Reply:
Make sense :)